ui/src/criteria/attribute.tsx

```diff
@@ -231,6 +232,21 @@ function AttributeInline(props: AttributeInlineProps) {
     );
   }
 
+  useEffect(() => {
+    if (attribute.dataType === tanagraUnderlay.SZDataType.BOOLEAN) {
+      updateCriteria(
+        produce(decodedData, (data) => {
+          data.selected = [
+            {
+              name: "",
+              value: true,
+            },
+          ];
+        })
+      );
+    }
+  }, []);
```

Comment:
Not sure if this is the best way to set the value but didn't have access to the `dataType` of the plugin in the `generateFilter` function.

Set `name` as an empty string so it doesn't get rendered along with the title.

Reply:
It is available, but the linter complains if the parameters are unused. You can just add them back. See `cohort.ts:159` and `criteria/classification.tsx:179`. This cause the criteria to to be changed and resaved.

Reply:
Nice, good to know. Moved this to `generateFilter`